settings.php

```diff
@@ -180,8 +180,7 @@ function page_optimize_settings_init() {
 add_action( 'admin_init', 'page_optimize_settings_init' );
 
 function page_optimize_add_plugin_settings_link( $plugin_action_links, $plugin_file = null ) {
-	$is_this_plugin = dirname( $plugin_file ) === basename( __DIR__ );
-	if ( ! $is_this_plugin ) {
+	if ( ! ( 'page-optimize/page-optimize.php' === $plugin_file && current_user_can( 'manage_options' ) ) ) {
```

Comment:
Random thought. Do you find it easier to read like this or when you do it with `||`?

Reply:
I actually wrote the `||` form first but felt like this was easier to read as "if it's not both of these positive things".

I'm not passionate about it either way though.

Reply:
A bit of bikeshedding...

The main difference between the two for me is:
- in the || version you can read the conditions one by one and they make sense regardless of order
- in the && version you HAVE to read them both, then negate

Fwiw, I'd even write 2 separate IFs that return early. Ok, <bikeshedding />. 😅

Reply:
Ah, yeah, two separate IFs is probably better. I can update :)